app/tests/src/test_ingest_bem_pdfs.py

```diff
@@ -75,65 +79,67 @@ def test__ingest_bem_pdfs(caplog, app_config, db_session, policy_s3_file, file_l
             _ingest_bem_pdfs(db_session, "/app/tests/src/util/", doc_attribs, save_json=False)
         else:
             _ingest_bem_pdfs(db_session, policy_s3_file, doc_attribs, save_json=False)
+        db_session.commit()
```

Comment:
For my own understanding, why did this get added?

Reply:
It's not needed and is removed. The correct fix was to not rely on the chunk ordering returned by `document.chunks`.